docs/submitting-applications.md

```diff
@@ -44,7 +44,7 @@ Some of the commonly used options are:
 * `--class`: The entry point for your application (e.g. `org.apache.spark.examples.SparkPi`)
 * `--master`: The [master URL](#master-urls) for the cluster (e.g. `spark://23.195.26.187:7077`)
 * `--deploy-mode`: Whether to deploy your driver on the worker nodes (`cluster`) or locally as an external client (`client`) (default: `client`) <b> &#8224; </b>
-* `--conf`: Arbitrary Spark configuration property in key=value format. For values that contain spaces wrap "key=value" in quotes (as shown).
+* `--conf`: Arbitrary Spark configuration property in key=value format. For values that contain spaces wrap "key=value" in quotes (as shown). Multiple configurations shall be passed individually.(e.g --conf <key>=<value> --conf <key>=<value>)
```

Comment:
nit: `key2=value2` for second option

Reply:
also put `--conf <key>=<value> --conf <key>=<value` in backtick

Reply:
And you're missing a space after period, period after g, and the second sentence is all parenthetical. For minor changes, please do proofread them. Just: "Multiple configurations should be passed as separate arguments. Example: `--conf key1=value1 --conf key2=value2`.

Reply:
Thanks all for review, will update the section as per the suggestions .